docs/platform/webcontainers/browser-support.md

```diff
@@ -64,6 +64,8 @@ We have beta support for Firefox. Please try it and [share feedback with us](htt
 
 Firefox does not fully support the required mode for cross-origin isolation, so you might encounter limitations when **running a server in a preview frame** within the StackBlitz editor. Third-party assets might get blocked due to the limitations of cross-origin isolation policies. However, you can work around this by opening your server preview in a separate window.
 
+If you are using Firefox Private Browsing, it's not possible to run a webserver. The reason is that we need to be able to install a ServiceWorker which is not possible in a private window.
```

Comment:
```suggestion
If you are using Firefox Private Browsing, it's not possible to run a web server. The reason is that we need to be able to use a Service Worker, which Firefox does not allow in Private Browsing windows.
```

A couple tweaks:

- “Service Worker” and “web server” are usually separate words, rather than compound words.

- Maybe avoid “install” when referring to service workers? While technically true (and nothing to be ashamed of!), I’m worried that users less accustomed to the spec's technical terms would understand it as installing software on their computer outside of the browser sandbox (like viruses and spyware, basically).

- Maybe use “Private Browsing windows” (the feature name) instead of “private windows” (a generic use of the adjective “private”), because we don’t want to suggest that users have no privacy when using normal browsing windows. Normal browsing windows with Firefox’s Enhanced Tracking Protection (enabled by default) and/or ad and tracker blocker extensions are already a pretty good privacy-protecting layer.